flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/catalog/CatalogStatisticsTest.java

```diff
@@ -61,10 +62,16 @@
 	public void testGetStatsFromCatalog() throws Exception {
 		EnvironmentSettings settings = EnvironmentSettings.newInstance().useBlinkPlanner().inBatchMode().build();
 		TableEnvironment tEnv = TableEnvironment.create(settings);
-		tEnv.registerTableSource("T1", new TestTableSource(true, tableSchema));
-		tEnv.registerTableSource("T2", new TestTableSource(true, tableSchema));
 		Catalog catalog = tEnv.getCatalog(tEnv.getCurrentCatalog()).orElse(null);
 		assertNotNull(catalog);
+		catalog.createTable(
+			ObjectPath.fromString("default_database.T1"),
```

Comment:
side comment: we should get rid of this method, it doesn't even support the identifier escaping